install.js

```diff
@@ -123,7 +124,7 @@ function read_file(success_cb, error_cb) {
 
 function download_file(callback) {
     var timeouthandle = null;
-    console.log("%s %s:%d%s", unicodedatafile.method, unicodedatafile.host,
+    console.log("%s %s://%s:%d%s", unicodedatafile.method, unicodedatafile.scheme, unicodedatafile.host,
```

Comment:
this also needs to be added below within `if (proxyServer)... unicodedatafile.path = unicodedatafile.scheme + "://" + unicodedatafile.host  `

this would then resolve many issues on dependant projects:
https://github.com/dodo/node-slug/issues/58
https://github.com/keystonejs/generator-keystone/issues/129

the symptom I'm seeing is `Error: Cannot find module 'unicode/category/So'`